content/2015.11.shweta-oak.rst

```diff
@@ -0,0 +1,80 @@
+Discovery of Kinto servers
+###########################
+
+:slug: outreachy-2016
+:date: 2015-11-27
+:lang: en
+:url: shweta-oak-kinto-2016
+:summary:
+    Work on Kinto project - making instances discoverable, during internship.
+
+.. epigraph::
+	
+  By having a program targeted specifically towards women, we found that we
+  reached talented and passionate participants, who were uncertain about how to
+  start otherwise.
+
+  We are now expanding the program to people from more groups underrepresented
+  in FOSS. We hope this effort will help many people learn how exciting, varied
+  and valuable work on FOSS projects can be and how inclusive the community
+  really is.
+
+  -- `The GNOME Outreachy program wiki <https://wiki.gnome.org/Outreachy>`_
+
+While looking for projects to contribute to, I read about Kinto, and found it intriguing.
```

Comment:
You need to introduce yourself first! Otherwise people won't know who they're reading :-)